### PR TITLE
Create a new content type (FAQ Entry)

### PR DESCRIPTION
**Pull request by Bakkhos**
Create a new CMS content type 'FAQ entry'

- Create a new folder collection in [static/admin/config.yml](https://github.com/hardforkio/homepage/blob/2c03f8d2469462c536ca15f3dd134aa7832ecf43/static/admin/config.yml#L35-L51)
- Create an index.tsx file for the collection that
  - [Loads the collection as a constant array](https://github.com/hardforkio/homepage/blob/2c03f8d2469462c536ca15f3dd134aa7832ecf43/src/data/faqEntry/index.tsx#L11)
  - Provides a useFAQEntry [hook](https://github.com/hardforkio/homepage/blob/2c03f8d2469462c536ca15f3dd134aa7832ecf43/src/data/faqEntry/index.tsx#L16) that returns the list of faq entries for the current locale, depending on locale context
- Also create two entries in the collection

Related to #118 
Also this PR is an example of how you can add a new (non-singleton) content type.

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -32,3 +32,20 @@ collections:
...
+        label: "title"
+        description: "A name to identify this object within this collection. Should be unique within the collection."
+        widget: "string"
+      - name: "translations"
+        label: "translations"
+        widget: "list"
+        fields:
+          - { name: "locale", widget: "select", options: ["en", "de"] }
+          - { name: "question", widget: "string" }
+          - { name: "answer", widget: "string" }
```
Comment (Bakkhos) [inline static/admin/config.yml]: Documentation on widgets (=field types): https://www.netlifycms.org/docs/widgets/#default-widgets

File: static/admin/config.yml [hunk 2]
```diff
@@ -32,3 +32,20 @@ collections:
               - { name: "subHeadline", widget: "string" }
               - { name: "metaTitle", widget: "string" }
               - { name: "content", widget: "markdown" }
+  - name: "faqEntry"
+    label: "FaqEntry"
+    folder: "src/data/faqEntry"
+    create: true
+    format: "json"
```
Comment (Bakkhos) [inline static/admin/config.yml]: Documentation for these options: https://www.netlifycms.org/docs/configuration-options/#collections

File: static/admin/config.yml [hunk 3]
```diff
@@ -32,3 +32,20 @@ collections:
...
+    fields:
```
Comment (Bakkhos) [inline static/admin/config.yml]: This declares a collection (=content type) named `faqEntry`, which is a so called [folder collection](https://www.netlifycms.org/docs/collection-types/#folder-collections). Each item is an object with the specified `fields`. One json file will be created in `src/data/faqEntrỳ` per content item.

These options are documented [here](https://www.netlifycms.org/docs/configuration-options/#collections)

File: src/data/faqEntry/asd.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (kiliw) [inline src/data/faqEntry/asd.json]: Shouldn't be this file removed?

File: src/data/faqEntry/der-5-hour-workday.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (kiliw) [inline src/data/faqEntry/der-5-hour-workday.json]: This file should not be in this PR or?

File: src/components/Navbar/MenuEntries.tsx [hunk 6]
```diff
@@ -1,14 +1,25 @@
...
+}) => (
   <>
     {[
       <NavItem key="contact">
         <NavLink to="/#contact" tag={linkTag}>
           KONTAKT
         </NavLink>
       </NavItem>,
+      <NavItem key="faq" className={showFAQ ? '' : 'd-none'}>
+        <NavLink to="/de/faq" tag={linkTag}>
```
Comment (kiliw) [inline src/components/Navbar/MenuEntries.tsx]: We should add a localized link here and not default to the german version.

**Comment by Levino** [comment]
@kiliw please fix conflicts.

**Pull request merged**